Cakefile

```diff
@@ -35,6 +35,11 @@ build = (callback) ->
             console.error err
         else
             build_webpack () ->
+                jsPath = build_dir + '/APIBlueprintImporter.js'
+                js = fs.readFileSync(jsPath, 'utf-8')
+                js = js.replace('var Module={"ready"', 'var Module={"ENVIRONMENT":"WEB","ready"')
```

Comment:
Note, I made this simple work around so I can continue with my task of using API Elements.